{"name": "test_error_app_key", "status": "passed", "description": "错误的app_key参数", "start": 1599987460097, "stop": 1599987460146, "uuid": "64764d14-674b-483e-8207-a45748459741", "historyId": "c65a9a8f558881df192a38af12688d4e", "testCaseId": "b02efde331dcd4e27dafc6ea28e8f72c", "fullName": "case.test_login_pytest.Test_Login_user#test_error_app_key", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "Test_Login_user"}, {"name": "host", "value": "V_LFJJLIU-NB0"}, {"name": "thread", "value": "22832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login_pytest"}]}